### ISSUE TITLE
Check for deprecated calls OC 8.0+

### ISSUE THREAD

**Issue by larsnaesbye** [deic-dk/user_orcid#3]

**Comment by larsnaesbye** [comment]
```
$user_id = OCP\USER::getUser(); //FIXME: Deprecated since 8.0.0 Use \OC::$server->getUserSession()->getUser()->getUID()
```

$query  = \OCP\DB::prepare($sql); //FIXME: Deprecated since 8.1.0 use prepare() of \OCP\IDBConnection - \OC::$server->getDatabaseConnection()

OCP\JSON::success(array('orcid' => $result)); //FIXME: deprecated in 8.1: Use a AppFramework JSONResponse instead

**Comment by larsnaesbye** [comment]
getUser was fixed.

Remaining calls: 

$query = \OCP\DB::prepare($sql); //FIXME: Deprecated since 8.1.0 use prepare() of \OCP\IDBConnection - \OC::$server->getDatabaseConnection()

OCP\JSON::success(array('orcid' => $result)); //FIXME: deprecated in 8.1: Use a AppFramework JSONResponse instead

**Comment by larsnaesbye** [comment]
Giving up on these - some of the functionality is not available in 7.0.x anyway.
